app/screens/home/search/results/file_card.tsx

```diff
@@ -0,0 +1,144 @@
+import {View, Text, TouchableOpacity} from 'react-native';
+
+import CompassIcon from '@components/compass_icon';
+import FormattedDate from '@components/formatted_date';
+import FileIcon from '@components/post_list/post/body/files/file_icon';
+import {useTheme} from '@context/theme';
+import {getFormattedFileSize} from '@utils/file';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+type Props = {
+    fileInfo: FileInfo;
+}
+
+const hitSlop = {top: 24, bottom: 24, left: 24, right: 24};
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
+    return {
+        container: {
+            flex: 1,
+            flexDirection: 'row',
+
+            marginHorizontal: 20,
+            marginVertical: 4,
+            paddingVertical: 8,
+
+            borderWidth: 1,
+            borderRadius: 4,
+            borderColor: changeOpacity(theme.centerChannelColor, 0.16),
+
+            shadowColor: changeOpacity(theme.centerChannelColor, 0.16),
+            shadowOffset: {
+                width: 0,
+                height: 2,
+            },
+            shadowOpacity: 0.8,
+            elevation: 1,
+        },
+        flexColumn: {flexDirection: 'column'},
+        flexRow: {flexDirection: 'row'},
+        iconContainer: {
+            marginHorizontal: 8,
+        },
+        nameText: {
+            color: theme.centerChannelColor,
+            ...typography('Heading', 200, 'SemiBold'),
+        },
+        infoText: {
+            color: changeOpacity(theme.centerChannelColor, 0.64),
+            ...typography('Body', 75, 'Regular'),
+        },
+        channelText: {
+            marginRight: 4,
+            backgroundColor: changeOpacity(theme.centerChannelColor, 0.08),
+            borderRadius: 4,
+            paddingHorizontal: 4,
+        },
+        dateText: {
+            marginBottom: 2,
+        },
+        threeDotContainer: {
+            flex: 1,
+            alignItems: 'flex-end',
+        },
+        threeDot: {
+            top: 13,
+            right: 20,
+        },
+    };
+});
+
+export default function FileCard({fileInfo}: Props) {
+    const theme = useTheme();
+    const style = getStyleSheet(theme);
+    const size = getFormattedFileSize(fileInfo.size);
+    const format = 'MMM DD HH:MM A';
+
+    // TODO: hook this up
+    const openGallery = () => {
+        /* eslint-disable no-console */
+        console.log('Open Gallery');
+    };
+
+    // TODO: hook this up
+    const handleThreeDotPress = () => {
+        /* eslint-disable no-console */
+        console.log('pressed 3 dot');
+    };
+
+    // TODO: Get the channel name and display
+    const channelName = () => { // eslint-disable-line @typescript-eslint/no-unused-vars
+        return 'channelName';
+    };
+
+    // TODO: Get the filetype and display
+    const fileType = () => { // eslint-disable-line @typescript-eslint/no-unused-vars
+        return 'fileType';
```

Comment:
is this the file type or extension ?

Reply:
My assumption is that this should not be the extension because the filename will include that information in its name. Note also that this value that is shown in figma does not exist in the webapp.  

@matthewbirtch, can you provide inside into the addition of this text and what do we want to show? It seems a bit redundant, unless the filename does not have an extenstion and the memetype is extracted from the file? What are you thoughts?

Reply:
This is intended be the file extension, not the filetype. I'd actually be ok with removing the extension element since it will most likely be showing the filename itself. This might help with the number of elements in that 2nd row too.